pandas/core/frame.py

```diff
@@ -1984,7 +1984,10 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
         Parameters
         ----------
         fname : str
```

Comment:
side issue. we use ``path`` elsewhere for IO routines. We should change this as well (out of scope here). would have to deprecate (the name) unfortunately.

Reply:
we actually use path on the top-level ``.to_parquet``, not sure how this is named this way.